web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php

```diff
@@ -253,6 +280,13 @@ protected function parseProfile(Profile $profile, array $columns) {
           ]);
           break;
 
+        // Generate community profile link based on a pattern defined on the
+        // settings page.
+        case 'community_link':
+          $community_site_url = Url::fromUri($this->urlGenerator->generate($profile));
+          $row[] = Link::fromTextAndUrl($this->t('Biblo.dk'), $community_site_url);
```

Comment:
@kasperg I write "Biblo.dk" as my text for the link to the front end here. Should we agree on some sort of standard? I can see we use `See on community site` on Flagged Content